### PR TITLE
Add Support for Microsoft Ads API

### PR DESCRIPTION
**Pull request by matthewbelair**
## Describe your changes

- Added configuration for microsoft-ads in providers.yaml
- Added microsoft-ads to docs-v2.
- Added microsoft-ads pages for Pre-Configured APIs group in mint.json.
- Consolidate microsoft-ads docs pages

## Test

This provider has been used successfully in local development to connect to Microsoft Ads and successfully call those endpoints after authorizing using Nango.

The documentation update was reviewed using npm run docs and verifying on localhost.

## Issue ticket number and link

Related to [this thread](https://nango-community.slack.com/archives/C04ABR352H0/p1712349133436789) in the Slack channel.

### CODE REVIEW

**Review by TBonnin — COMMENTED** [review]
Thank you @matthewbelair It should probably be added to https://github.com/NangoHQ/nango/blob/2e9d5e7e52ffc4ef81988dd94661538fc31b0e32/docs-v2/integrations/marketing.mdx?plain=1#L9

**Comment by matthewbelair** [comment]
@TBonnin sure, done. Any timeline for when this can be merged? Thanks

**Comment by TBonnin** [comment]
> @TBonnin sure, done. Any timeline for when this can be merged? Thanks

Can you rebase your branch? Then I will merge it before the end of the day

File: packages/shared/providers.yaml [hunk 1]
```diff
@@ -1321,6 +1321,18 @@ microsoft-tenant-specific:
...
         grant_type: refresh_token
     docs: https://docs.nango.dev/integrations/all/microsoft-tenant-specific
+microsoft-ads:
+    alias: microsoft-teams
+    categories:
+        - marketing
+    default_scopes:
+        - https://ads.microsoft.com/msads.manage
+        - offline_access
+    proxy: null
```
Comment (TBonnin) [inline packages/shared/providers.yaml]: `proxy: null` is not allowed actually. Just remove this line, then the proxy value of the aliased provider will be used. Sorry I should have picked this up earlier

**Review by TBonnin — CHANGES_REQUESTED** [review]

**Comment by matthewbelair** [comment]
@TBonnin should be good now. Thanks!

**Pull request merged**